lib/build/clang

```diff
@@ -4,22 +4,25 @@ set -e
 p="$@"
 set --
 
+CLANG=${CLANG:='clang-12'}
+crtBeginXlinker="$CRT_BEGIN_XLINKER"
+crtEndXlinker="$CRT_END_XLINKER"
```

Comment:
Don't love this either, see below for a longer discussion on how we can hopefully get `crt{begin/end}` working without us having to explicitly specify anything in our wrapper scripts.